openstack/identity/v3.0/security/GetOperationProtectionPolicy.go

```diff
@@ -7,7 +7,17 @@ import (
 
 type ProtectionPolicy struct {
 	// Indicates whether operation protection has been enabled. The value can be true or false.
-	OperationProtection *bool `json:"operation_protection" required:"true"`
+	OperationProtection *bool `json:"operation_protection"`
+	// Specifies whether a person is designated for verification.
+	AdminCheck string `json:"admin_check"`
```

Comment:
should add omitempty to optional params to avoid passing zero values

Reply:
this struct only for response